_posts/2017/05/2017-05-15-EmpoweringOthers.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+authors: ["Rayna Harris"]
+title: "What is the reward for empowering others?"
+date: 2017-05-14
+time: "20:00:00"
+category: ["Community","Teaching","Education","Workshops","Mentoring"]
+---
+
+What is the reward for empowering others? Following a few round-table discussions with Software Carpentry instructors, I gained some insight into what drives so many of us to volunteer our time so enthusiastically. I came to the conclusion that many of us share a sense of accomplishment when we empower others through mentoring and education. Software Carpentry gives us a fountain-full of opportunities to contibute through community developed lessons, community supported workshops, and world-wide community conversations. We start out volunteering a little when there i time, but we soon carve out more and more space in our schedule for Software Carpentry events and activities that provide the opportunity to empower others.  
```

Comment:
I would remove the first sentence (question) to improve the flow. 
"i time" -> "is time"
Thanks for writing this!